server/database/models/getDataUsingEmail.js

```diff
@@ -0,0 +1,4 @@
+const { User } = require('./user');
+
+const getEmail = (userEmail) => User.findOne({ where: { email: userEmail } });
```

Comment:
all we're doing here is to call findOne from user, so there is no need to create  a separate function for it, 
In the place you want to find a user call your method there